SPECS/xorg-x11-font-utils/xorg-x11-font-utils.spec

```diff
@@ -1,23 +1,21 @@
 %global bdftopcf 1.1
-%global fonttosfnt 1.2.1
-%global mkfontdir 1.0.7
-%global mkfontscale 1.1.3
-%global font_util 1.3.1
+%global fonttosfnt 1.2.3
+%global mkfontscale 1.2.2
+%global font_util 1.4.1
 # Must be kept in sync with xorg-x11-fonts!
 %global _x11fontdir %{_datadir}/X11/fonts
 Summary:        X.Org X11 font utilities
 Name:           xorg-x11-font-utils
 Version:        7.5
-Release:        50%{?dist}
+Release:        51%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://www.x.org
 Source0:        https://www.x.org/pub/individual/app/bdftopcf-%{bdftopcf}.tar.bz2
-Source1:        https://www.x.org/pub/individual/app/fonttosfnt-%{fonttosfnt}.tar.bz2
-Source2:        https://www.x.org/pub/individual/app/mkfontdir-%{mkfontdir}.tar.bz2
-Source3:        https://www.x.org/pub/individual/app/mkfontscale-%{mkfontscale}.tar.bz2
-Source4:        https://www.x.org/pub/individual/font/font-util-%{font_util}.tar.bz2
+Source1:        https://www.x.org/pub/individual/app/fonttosfnt-%{fonttosfnt}.tar.xz
+Source3:        https://www.x.org/pub/individual/app/mkfontscale-%{mkfontscale}.tar.xz
```

Comment:
do we want the numbering to be like this? this could be confusing to future reader